spec/models/vacancy_spec.rb

```diff
@@ -250,4 +250,54 @@
       end
     end
   end
+
+  context 'Content sanitization' do
+    it '#job_description' do
```

Comment:
There's potential to put these similar methods into a shared example to DRY this up a bit.